index.d.ts

```diff
@@ -10,7 +10,7 @@ declare const leven: {
 	//=> 2
 	```
 	*/
-	(left: string, right: string): number;
+	(left: string, right: string, options?: any): number;
```

Comment:
The options should be properly typed and documented here. See: https://github.com/sindresorhus/typescript-definition-style-guide#documentation